soap/src/java/com/zimbra/soap/mail/type/ActionSelector.java

```diff
@@ -153,6 +153,13 @@
     protected String tagNames;
 
     /**
+     * @zm-api-field-tag non-existent-ids
+     * @zm-api-field-description Comma-separated list of non-existent ids
+     */
+    @XmlAttribute(name=MailConstants.A_NON_EXISTENT_IDS /* nei */, required=false)
+    protected String nonExistentIds;
```

Comment:
Meaning right now it's always present in the response if it's a HARD_DELETE that is requested.